### ISSUE TITLE
Website Accessibility-WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (.indented:nth-child(5))

### ISSUE THREAD

**Issue by A11Ybot-OP** [SFTtech/openage#1446]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[openage - free, modern RTS game engine](https://openage.sft.mx/)
#### Element path
.indented:nth-child(5)
#### Snippet
`<p class="indented"><b>Multiplayer:</b> We implement a client-server architecture with dedicated servers that supports more than 8 players. The overarching system will provide matchmaking, lobbies, server discovery and other community features.</p>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 1.37 (foreground color: #dcdcdc, background color: #ffffff, font size: 13.3pt (17.687px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Microsoft Edge version 96.0.1054.62
This accessibility issue was found using Accessibility Insights for Web 2.29.0 (axe-core 4.3.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
